docs/api/orders.rst

```diff
@@ -3,165 +3,88 @@ Orders API
-                                "sku":"8808"
-                            },
-                            "inventory_units":[
-                            ],
-                            "_links":{
-                                "product":{
-                                    "href":"\/api\/products\/2107"
-                                },
-                                "variant":{
-                                    "href":"\/api\/products\/2107\/variants\/13099"
-                                }
-                            }
-                        }
-                    ],
-                    "items_total":97783,
-                    "adjustments":[
-                    ],
-                    "comments":[
+.. code-block:: bash
 
-                    ],
-                    "adjustments_total":24240,
-                    "total":122023,
-                    "confirmed":true,
-                    "created_at":"2014-04-30T10:41:14+0000",
-                    "updated_at":"2014-11-26T23:00:34+0000",
-                    "state":"pending",
-                    "email":"ygrant@example.com",
-                    "expires_at":"2014-11-27T02:00:33+0000",
-                    "user":{
-                        "id":476,
-                        "username":"ygrant@example.com",
-                        "username_canonical":"ygrant@example.com",
-                        "email":"ygrant@example.com",
-                        "email_canonical":"ygrant@example.com",
-                        "enabled":false,
-                        "group":[
+    $ curl http://demo.sylius.org/api/v1/orders/21 \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json"
 
-                        ],
-                        "locked":false,
-                        "expired":false,
-                        "roles":[
+.. note::
 
-                        ],
-                        "credentials_expired":false
-                    },
-                    "channel":{
-                        "id":91,
-                        "code":"WEB-UK",
-                        "name":"UK Webstore",
-                        "type":"web",
-                        "color":"Red",
-                        "enabled":true,
-                        "created_at":"2014-11-26T23:00:15+0000",
-                        "updated_at":"2014-11-26T23:00:15+0000",
-                    },
-                    "shipping_address":{
-                    },
-                    "billing_address":{
-                    },
-                    "payments":[
-                    ],
-                    "shipments":[
-                    ],
-                    "currency":"GBP",
-                    "checkout_state":"cart"
-                }
-            ]
-        }
-    }
+    The value *21* was taken from previous responses, where we managed a cart. Your value can be different. If you need more information about cart api please, check `this doc`__
```

Comment:
While referencing docs use the :doc:`this documentation <\directory\document>` format :)